WooCommerce/Classes/ViewRelated/Dashboard/MyStore/NewOrdersViewController.swift

```diff
@@ -60,6 +60,22 @@ extension NewOrdersViewController {
 
         StoresManager.shared.dispatch(action)
     }
+
+    func syncOrderStatus(onCompletion: ((Error?) -> Void)? = nil) {
+        guard let siteID = StoresManager.shared.sessionManager.defaultStoreID else {
+            onCompletion?(nil)
+            return
+        }
+
+        let action = OrderStatusAction.retrieveOrderStatuses(siteID: siteID) { (orderStatuses, error) in
```

Comment:
Ok, so: here is where I create the action. Below, I dispatch the action. BUT, it never reaches the actual OrderStatusStore function when I place breakpoints. I'm not sure why the function doesn't fire after the dispatch(action) happens.